renovate.json

```diff
@@ -5,6 +5,10 @@
     "prConcurrentLimit": 0,
     "pruneBranchAfterAutomerge": true,
     "rpmVulnerabilityAutomerge": "ALL",
+    "addLabels": [
+        "/approve",
+        "/lgtm"
```

Comment:
any chance the docs are not 100% accurate and we have to put labels without the leading `/`?